recipes/cutadapt/meta.yaml

```diff
@@ -1,17 +1,20 @@
+{% set name = "cutadapt" %}
 {% set version = "4.7" %}
 
 package:
-  name: cutadapt
+  name: {{ name|lower }}
```

Comment:
This is unneeded because the name of the package is already lowercase. Also, the name of the package is always going to be `cutadapt`, so there is no need to use a variable.